gui/builtinViews/fittingView.py

```diff
@@ -339,7 +339,11 @@ def removeItem(self, event):
     def removeModule(self, module):
         sFit = service.Fit.getInstance()
         fit = sFit.getFit(self.activeFitID)
-        populate = sFit.removeModule(self.activeFitID, fit.modules.index(module))
+        try:
+            populate = sFit.removeModule(self.activeFitID, fit.modules.index(module))
+        except ValueError:
+            # This module isn't in our list of modules, don't remove anything. Likely a special snowflake.
+            populate = None
```

Comment:
it's there for double click (also, sanity check in case anything else gets added that isn't a module). I'll check and make sure modes don't have a remove option in the submenu. 

I think that we need it regardless, it fails to the state we want (not doing anything), we are just trapping the stacktrace here.